Common.mk

```diff
@@ -142,12 +144,15 @@ IMAGE_TARGETS=$(foreach image,$(IMAGE_NAMES),$(if $(filter true,$(BUILD_OCI_TARS
 
 #################### HELM ##########################
 HAS_HELM_CHART?=false
-HELM_SOURCE_REPOSITORY?=$(REPO_OWNER)/$(REPO)
+HELM_SOURCE_OWNER?=$(REPO_OWNER)
+HELM_SOURCE_REPOSITORY?=$(REPO)
 HELM_GIT_TAG?=$(GIT_TAG)
 HELM_DIRECTORY?=.
-HELM_REPOSITORY?=$(REPO)
-HELM_GIT_CHECKOUT_TARGET?=$(HELM_REPOSITORY)/eks-anywhere-checkout-$(HELM_GIT_TAG)
-HELM_GIT_PATCH_TARGET?=$(HELM_REPOSITORY)/eks-anywhere-helm-patched
+HELM_DESTINATION_REPOSITORY?=$(IMAGE_COMPONENT)
+HELM_ADDITIONAL_KEY_VALUES?=
+HELM_GIT_CHECKOUT_TARGET?=$(HELM_SOURCE_REPOSITORY)/eks-anywhere-checkout-$(HELM_GIT_TAG)
+HELM_GIT_PATCH_TARGET?=$(HELM_SOURCE_REPOSITORY)/eks-anywhere-helm-patched
+PUBLIC_REGISTRY=$(shell aws ecr-public describe-registries --region us-east-1 --query 'registries[*].registryUri' --output text)
```

Comment:
Can we pass this in via the build instead?  Doing this kind of things make its difficult to run/test these things locally.  Why is it different than IMAGE_REPO?

Reply:
Right now, we have no mechanism to helm install out of a private repo, so I made this change so we can build the helm chart in private, but it will pull images from public. I'll change to allow override here though.